CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Added
 - Added a page about migrating potentially breaking changes to the documentation
 - markdown_include in development requirements file
+- STR variants filter
+- Display source, Z-score, inheritance pattern for STR annotations from Stranger (>0.6.1) if available
```

Comment:
Do we have a case with strs annotations from Stranger on stage that I could check?

Reply:
I don't think so; the demo case sort of covers it, but we should get that going with the MIP folks as well. The most important thing on stage is making sure old cases dont break - and decide if it is too confusing that the filter is there but not really working for the old ones.